### PR TITLE
Update Payment Methods

### PR DESCRIPTION
**Pull request by boldsuck**
- Tether icon added
- TransferWise-USD  added
- Account limits & signing added
- Backup & Restore added
- Image captions & Image light and dark mode added in mkdocs.yml
- Tables, links and spelling fixed

### CODE REVIEW

**Comment by boldsuck** [comment]
Just as a preview. 

Have I understood account aging correctly? (30, 60, 90 days)
Sources are linked in the documentation
https://boldsuck.github.io/haveno-docs/the-project/account_limits/

File: docs/the-project/payment_methods/0-all-methods.md [hunk 1]
```diff
@@ -10,12 +10,13 @@ Unlike cryptocurrency transfers, many fiat payment transfers are riddled with fl
 
 The payment methods vary in chargeback risk, regional availability, transaction size, fees, privacy, verifiability, and other characteristics. The top consideration for maintaining payment methods is chargeback risk.
 
-We welcome suggestions for new payment methods on Haveno, especially for those that enable new markets. Documentation on the criteria and process for adding new payment methods is found [here](https://github.com/haveno-dex/listing). Please also feel free to make suggestions by reaching out on Matrix room **Haveno Development** [#haveno-dev:monero.social](https://matrix.to/#/#haveno-dev:monero.social) relayed on IRC/Libera (`#haveno-dev`)
+We welcome suggestions for new payment methods on Haveno, especially for those that enable new markets. Documentation on the criteria and process for adding new payment methods is found [here](https://github.com/haveno-dex/listing). Please also feel free to make suggestions by reaching out on Matrix room **Haveno Development** [#haveno-development:monero.social](https://matrix.to/#/#haveno-development:monero.social) relayed on IRC/Libera (`#haveno-development`)
```
Comment (woodser) [inline docs/the-project/payment_methods/0-all-methods.md:13]: This change is already made in https://github.com/haveno-dex/haveno/pull/1423

File: docs/the-project/payment_methods/0-all-methods.md [hunk 2]
```diff
@@ -10,12 +10,13 @@ Unlike cryptocurrency transfers, many fiat payment transfers are riddled with fl
...
 
 Below is a list of fiat payment methods Haveno currently supports.
 
 !!! note
     The maximum trade sizes listed below are not available for most newly-created payment accounts. Please refer to Account signing for details on how to enable larger trade sizes for your payment accounts.
+    **Haveno v1.0.15 will increase some limits: Pay by Mail to 25 XMR and all Altcoin to 500 XMR**
```
Comment (woodser) [inline docs/the-project/payment_methods/0-all-methods.md:19]: These values are inconsistent with https://github.com/haveno-dex/haveno/pull/1424, which raises the crypto limit to 528 XMR, and may by mail to 48 XMR.

Comment (boldsuck) [reply docs/the-project/payment_methods/0-all-methods.md:19]: Danm, don't trust verify. ;-) They posted the wrong numbers in SimpleX chat.
I've changed it. When .15 is released, I will revise the table overview.

File: docs/the-project/payment_methods/0-all-methods.md (hunk 1 above)
Comment (boldsuck) [reply docs/the-project/payment_methods/0-all-methods.md:13]: Yeah, in README.md & CONTRIBUTING.md
I have adjusted the link in Payment Methods/0-all-methods.md.

**Pull request closed without merging**